modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -128,46 +135,48 @@ returns an empty matrix ( Mat::data==NULL ). Currently, the following file forma
     codecs supplied with an OS image. Install the relevant packages (do not forget the development
     files, for example, "libjpeg-dev", in Debian\* and Ubuntu\*) to get the codec support or turn
     on the OPENCV_BUILD_3RDPARTY_LIBS flag in CMake.
-
-@note In the case of color images, the decoded images will have the channels stored in B G R order.
- */
+@param filename Name of file to be loaded.
+@param flags Flag that can take values of cv::ImreadModes
+*/
 CV_EXPORTS_W Mat imread( const String& filename, int flags = IMREAD_COLOR );
 
 /** @brief Loads and resizes down an image from a file.
-@anchor imread_reduced
+
+The function imread_reduced loads an image from the specified file scaled 1/scale_denom and returns it.
+
+@note
+For jpeg images it uses the internal libjpeg facilities to get the reduced image. Thus valid scale_denom
```

Comment:
@mshabunin could you check this information ( maybe i failed to explain with my poor English) any better explanation is welcome.